### ISSUE TITLE
[digitizer] refer to featureType: with reference to a name is not possible

### ISSUE THREAD

**Issue by astroidex** [mapbender/mapbender#1337]
**Describe the bug**
Digitizer: Definition featuretype in config.yml und Verweis auf featureType name im Digitizer:
In 3.0.8.x war es möglich featureTypes in der config.yml zu definieren und auf diese im Digitizer zu verweisen. In 3.2.4 wird die Information aus der config.yml nicht genutzt.

It should be possible to refer to a featureType-name thatrefers to a configuration in config.yml 

Definition in config.yml

```
parameters:
  featureTypes:
    poi_feature_type:
      connection: geodata_db   
      table: public.poi   
      uniqueId: gid       
      geomType: point      
      geomField: geom   
      srid: 4326       
```

Definition in digitizer:

-> Veweis auf den Namen aus der config.yml:

`featureType: poi_feature_type`

-> statt der gesamten Konfiguration:
```
featureType: 
	connection: geodata_db
	table: poi
	uniqueId: gid
	geomType: point
	geomField: geom
	srid: 4326

```
**Expected behavior**
It should be possible to refer to a featureType-name thatrefers to a configuration in config.yml 

**Versions (please complete the following information if it helps to narrow down the bug):**
 - Mapbender: 3.2.4
 - 
**Screenshots / Logmessage**
![Bildschirmfoto vom 2021-04-01 15-06-33](https://user-images.githubusercontent.com/996298/113303766-b2444100-9301-11eb-9f9e-4d0e0fd602a7.png)

**Comment by thzanger** [comment]
Dear @astroidex, thank you for this information. I forwarded this issue into our internal ticket system.

**Comment by werrolf** [comment]
Fixed in [Digitizer 1.4.7](https://github.com/mapbender/mapbender-digitizer/releases/tag/1.4.7)

**Comment by astroidex** [comment]
works fine with Mapbender 3.2.5rc2 (tested with 1.4.8)
